WEB/Src/PerformanceCollector/Perf.Tests/Perf.Tests.csproj

```diff
@@ -0,0 +1,70 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'Test.props'))\Test.props" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), NetCore.props))\NetCore.props" />
+
+  <Import Project="$(PropsRoot)\Test.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;netcoreapp1.0;netcoreapp2.0</TargetFrameworks>
+    <AssemblyName>Microsoft.AI.PerformanceCollector.Tests</AssemblyName>
+    <PackageId>Microsoft.AI.DependencyCollector.Tests</PackageId>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <RootNamespace>Microsoft.ApplicationInsights.Tests</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="QuickPulse\PerfLib\PerfData.data" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.1.1" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.1.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp1.0'">
+    <PackageReference Include="NETStandard.HttpListener" Version="1.0.2" />
```

Comment:
This dependency was failing the build.

I'm aware of this issue, but I don't know why it's only just now causing failures.
I know what it's used for, it's providing some testing stubs some Http classes which weren't available yet in NetCore 1.0.
We're about to cut support for NetCore 1.x so instead of investigating a fix, you can delete this dependency and use `#IF NETCOREAPP1_0` and `[IGNORE("not supported in netcore 1.0"]` the specific tests that were depending on this.

Reply:
I made an inverted if and removed the dependency!